content/source/docs/cloud/run/cli.html.md

```diff
@@ -111,78 +98,31 @@ If you use a combined repository and [specify a working directory on workspaces]
 
 CLI-driven runs upload an archive of your configuration directory
 to Terraform Cloud. If the directory contains files you want to exclude from upload,
-you can do so by defining a [`.terraformignore` file in your configuration directory][remote].
+you can do so by defining a [`.terraformignore` file in your configuration directory](/docs/cli/using-terraform-cloud/initialization.html).
```

Comment:
This file doesn't exist and doesn't exist in the other [PR either](https://github.com/hashicorp/terraform/pull/29947)

Reply:
i think we can remove this link if the link doesn't exist tbh

Reply:
I have fixed this!